mobile_base_sdk/mobile_base_sdk.py

```diff
@@ -214,6 +214,7 @@ async def goto_async(
         self._mobility_stub.SendGoTo(req)
 
         tic = time.time()
+        arrived = False
```

Comment:
This variable is going to be directly set to True or is there a change we don't go inside the while?
I'd set it to True and remove the line 219.